inst/extdata/example_params.toml

```diff
@@ -39,8 +39,8 @@ p_hosp_w_sd_sd = 0.01
 
 inv_sqrt_phi_prior_mean = 0.1 # 1 / sqrt(100)
 inv_sqrt_phi_prior_sd = 0.1414214 # 1 / sqrt(50)
-wday_effect_prior_mean = 0.1428571 # 1 / 7
-wday_effect_prior_sd = 0.05
+hosp_wday_effect_prior_alpha = [5, 5, 5, 5, 5, 5, 5]
```

Comment:
This might be an incredibly naive question, but can you explain your choice of all 5s here? Is this a recommended non-informative prior choice for a direchlet?

Reply:
As I mentioned above, 
> Parameters chosen to make the Dirichlet prior similar to but less informative than the earlier constrained Normal.

And yes, for reasoning about the parameters you can use a similar [updating-based intuition](https://en.wikipedia.org/wiki/Dirichlet_distribution#Conjugate_to_categorical_or_multinomial) to the one for the Beta, just with more categories. So because all the numbers as the same (5), this prior will be centered on equal weekly reporting, just as the previous one was intended to be. But because none of the numbers are large, uneven reporting is a priori plausible. And of course, always good to check intuition via forward simulation (I did a bit of that as well).